portfolio-website/script.js

```diff
@@ -1,23 +1,79 @@
+// navBar
 const navBarList = [
-    "Ingredients", "Steps to Make", "Links to Other Wonton Soup Recipes", "About Me"
+    "About Me", "Projects", "Contact"
 ]
 
 for (let i = 0; i < navBarList.length; i++) {
     const label = document.createElement('a');
     label.innerText = navBarList[i];
     label.addEventListener('mouseover', function() {
-        label.style.color = "white";
+        label.style.color = "black";
     })
     label.addEventListener('mouseout', function() {
-        label.style.color = "black";
+        label.style.color = "white";
     })
     label.addEventListener('click', function() {
         label.setAttribute('href', `#${navBarList[i].split(" ").join("")}`);
-        if (i === 3) {
-            label.setAttribute('href', "./aboutme.html");
+        if (i === 2) {
+            label.setAttribute('href', "./contactForm.html")
         }
         label.style.textDecoration = "none";
     })
 
     document.getElementById("navBar").appendChild(label);
-}
\ No newline at end of file
+}
+// end navBar
+
+// typewriter functionality
+const span1 = document.createElement('span');
+span1.textContent = "Welcome to";
```

Comment:
we are re-writing the Welcome to and quoteArray, can we set this string to a variable and then re-use in both places? 

span2.textContent and same with quoteArray2 (but with quoteArray2 use string literals)